src/tsickle.ts

```diff
@@ -1870,6 +1890,18 @@ class ExternsWriter extends ClosureRewriter {
   }
 }
 
+function isPolymerBehaviorPropertyInCallExpression(pa: ts.PropertyAssignment): boolean {
+  const parentParent = pa.parent && pa.parent.parent;
+  if (pa.name.kind !== ts.SyntaxKind.Identifier ||
```

Comment:
I included the check for `parentParent` in this `if` clause (based on your comment).